tests/BasicRum/Beacon/Importer/Process/BeaconTest.php

```diff
@@ -19,6 +20,20 @@ public function testMissingProcessIdConstraints()
             ]
         ];
 
+        $decompressor = new ResourceTimingDecompressor_v_0_3_4();
```

Comment:
The previous tests were failing because we forgot to remove this code. It tries to simulate that 'restiming' was decoded in catcher.php but we discussed that we should not do this. restiming will be decoded on beacon import.